<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManPowerGui</class>
 <widget class="QWidget" name="ManPowerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="set_point_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>101</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_point_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="add_point">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>add point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_point">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>delete point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_piece">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>delete piece</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>delete frame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_polygon">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>create polygon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_mode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>change mode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
